theano/configparser.py

```diff
@@ -98,15 +98,17 @@ class change_flags(object):
 
     Useful during tests.
     """
-    def __init__(self, **kwargs):
+    def __init__(self, *args, **kwargs):
         confs = dict()
-        for k in kwargs:
+        args = dict(*args)
```

Comment:
I find it confusing to have `*args` here, but to require it to only be 1 argument at most (otherwise the call to `dict` fails).
Why not `def __init__(self, d=None, **kwargs):` and then `if not d: d = {}; else d = dict(d)`?
I won't hold the release for that, though.